includes/wikia/tasks/Tasks/ImageReviewTask.class.php

```diff
@@ -41,9 +41,12 @@ public function delete( $pageList, $suppress = false ) {
 			$cityLang = \WikiFactory::getVarValueByName( 'wgLanguageCode', $wikiId );
 			$reason = wfMsgExt( 'imagereview-reason', ['language' => $cityLang] );
 
-			$command = "SERVER_ID={$wikiId} php {$IP}/maintenance/wikia/deleteOn.php" .
-				' -u ' . escapeshellarg( $userName ) .
-				' --id ' . $imageId;
+			// TODO: community switch
+			//$command = "SERVER_ID={$wikiId} php {$IP}/maintenance/wikia/deleteOn.php" .
+			//	' -u ' . escapeshellarg( $userName ) .
+			//	' --id ' . $imageId;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)